pkg/machinery/constants/constants.go

```diff
@@ -445,6 +445,51 @@ const (
 
 	// DefaultSecondaryResolver is the default secondary DNS server.
 	DefaultSecondaryResolver = "8.8.8.8"
+
+	// WireguardDefaultPort is the default listen port for Wireguard interfaces.
+	WireguardDefaultPort = 51820
+
+	// WireguardDefaultRoutingTable is the default routing table for Wireguard LAN targets.
+	// This is only used for Wireguard LAN (WireguardAutoNodes), and it specifies the routing table which will be used for Wireguard-available destinations.
+	WireguardDefaultRoutingTable = 180
+
+	// WireguardDefaultFirewallMark is the default firewall mark to use for Wireguard encrypted egress packets.
+	// This default will not be applied unless it is required (for example, using WgLAN (WireguardAutoNodes)).
+	// The next consecutive mark will be used by WgLAN to mark packets which should be forced into the Wireguard interface.
+	// Normal Wireguard configurations will NOT use this firewall mark.
+	WireguardDefaultFirewallMark = 0x51820
+
+	// WireguardDefaultForceFirewallMark is the default firewall mark to use for packets destined to IPs serviced by Wireguard LAN.
+	// It is used to signal that matching packets should be forced into the Wireguard interface.
+	WireguardDefaultForceFirewallMark = 0x51821
+
+	// WireguardDefaultNodesPrefix is the default prefix to use for Wireguard Automatic Node addition.
+	WireguardDefaultNodesPrefix = "0200:1001::/64"
```

Comment:
Seán, I think you said it should be updated

Reply:
Yes, thank you:  needs to be updated to proper ULA address model.